### PR TITLE
Rubocop refactoring

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
Prompted by https://github.com/Homebrew/brew/pull/3876 but nothing big this time; all these changes make the code more readable.

### CODE REVIEW

File: Formula/envconsul.rb [hunk 1]
```diff
@@ -30,16 +30,12 @@ def install
     CONSUL_DEFAULT_PORT = 8500
     LOCALHOST_IP = "127.0.0.1".freeze
 
-    def port_open?(ip, port, seconds = 1)
+    def port_open?(ip_address, port, seconds = 1)
```
Comment (ilovezfs) [inline Formula/envconsul.rb]: I guess not needed if we're disabling the must be more than two letters thing.

File: Formula/fabio.rb [hunk 2]
```diff
@@ -33,16 +33,12 @@ def install
     FABIO_DEFAULT_PORT = 9999
     LOCALHOST_IP = "127.0.0.1".freeze
 
-    def port_open?(ip, port, seconds = 1)
+    def port_open?(ip_address, port, seconds = 1)
```
Comment (ilovezfs) [inline Formula/fabio.rb]: ditto

File: Formula/soci.rb [hunk 3]
```diff
@@ -33,9 +29,10 @@ def translate(a)
   def install
     args = std_cmake_args + %w[.. -DWITH_SQLITE3:BOOL=ON]
 
-    %w[boost mysql oracle odbc pg].each do |a|
-      bool = build.with?(a) ? "ON" : "OFF"
-      args << "-DWITH_#{translate(a).upcase}:BOOL=#{bool}"
+    %w[boost mysql oracle odbc pg].each do |arg|
```
Comment (ilovezfs) [inline Formula/soci.rb]: ditto

**Comment by MikeMcQuaid** [comment]
@ilovezfs On all the cases above I was willing to disable the three letter thing for taps but I do think these changes improve rather than harm readability.

**Comment by MikeMcQuaid** [comment]
> 🤷‍♂️

Assuming that means ✅  ( :trollface: )

**Pull request merged**